crates/api_common/src/oauth_provider.rs

```diff
@@ -72,6 +76,7 @@ pub struct DeleteOAuthProvider {
 /// Logging in with an OAuth 2.0 authorization
 pub struct AuthenticateWithOauth {
   pub code: String,
+  pub pkce_code_verifier: Option<String>,
```

Comment:
You need to add this to make ci pass:

```suggestion
  #[cfg_attr(feature = "full", ts(optional))]
  pub pkce_code_verifier: Option<String>,
```